### ISSUE TITLE
DXF file modification/editing.

### ISSUE THREAD

**Issue by rspardeshi** [haplokuon/netDxf#407]
Hi Haplokuon,

Is it possible to update, edit, or alter the DXF file data using netDxf ?
I want to select a particular Polyline and wish to delete that polyline from the selected DXF file.

thanks,
Rahul Pardeshi.

**Comment by haplokuon** [comment]
Yes, the the purpose of this library is being able to create, modify, and reading DXF files. You just need to load your file, remove the entity, and save it again.

**Comment by rspardeshi** [comment]
Hi Haplokuon,

thanks,
I am studying netDXF library and successfully created one tool using it.
Now I want to select a particular Polyline and wish to delete that polyline from the selected DXF file.
Could you please provide me any example or reference matching my task?

**Comment by haplokuon** [comment]
I cannot tell you how to find a particular entity in the drawing you must know how to find it. You will have to decide if it has a particular color, if it is inside a unique layer, if it is by its geometry,...; although the most efficient way is by its handle.
```c#
DxfObject o = doc.GetObjectByHandle("");
doc.Entities.Remove(o as EntityObject);
```

**Comment by rspardeshi** [comment]
thanks for the advice... it's constructive for my development.

**Comment by my88480** [comment]
List<Polylines2D> allOuterContourEnts = DxfDoc.Entities.Polylines2D.Where(e => e.Layer.Name.Equals("0")).ToList();
DxfDoc.Entities.Remove(allOuterContourEnts);
